src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -445,15 +445,15 @@ DEBUG:  push down of limit count: 30
  l_partkey | o_custkey | l_quantity 
 -----------+-----------+------------
     154380 |       421 |      26.00
-    103981 |       325 |      26.00
+    138854 |        79 |      26.00
```

Comment:
Since removing PRIMARY KEY constraints change the query plan, it effects the order of rows with the same l_quantity. I have changed the column used by ORDER BY to l_partkey, o_custkey from l_quantity to generate the same result regardless of the plan.